### ISSUE TITLE
[USE] Handling of Brahmic characters with ISC Other undefined

### ISSUE THREAD

**Issue by NorbertLindenberg** [MicrosoftDocs/typography-issues#470]
The USE description doesn’t say how Brahmic characters are interpreted for which the Unicode character data doesn’t define an Indic Syllabic Category, or whose ISC is the default value Other, and whose script is not Common. A few such characters in Unicode 13 are U+0971, U+ 0950, U+A8F4..U+A8F7, U+A8FB, U+A8FD, U+1CED, U+1CE2..U+1CE8. (Some of these characters probably should have different syllabic categories, and I’ve reported this issue also to the Unicode Consortium.)

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 194a6d3c-4137-46e9-3a4b-44b990200986
* Version Independent ID: a0c8e788-5228-aa28-670e-3ba1ac3faecd
* Content: [Creating and supporting OpenType fonts for the Universal Shaping Engine - Typography](https://docs.microsoft.com/en-us/typography/script-development/use)
* Content Source: [typographydocs/script-development/use.md](https://github.com/MicrosoftDocs/typography/blob/live/typographydocs/script-development/use.md)
* Product: **typography**

**Comment by xadxura** [comment]
@NorbertLindenberg, Good point. These default to BASE_IND. Certainly, better category assignments might be made for some of these.

**Comment by xadxura** [comment]
Document has been updated.

**Comment by NorbertLindenberg** [comment]
The documentation now says that all characters with UISC = Other go into BASE_IND. That doesn’t work because there are characters that have UISC = Other but have already been assigned to other classes for good reasons. One example are the Balinese musical symbols 1B61..1B6A and 1B74..1B7C, which have general category So and therefore USE class SYM, which allows the combining marks 1B6B..1B73 to attach to them.

The intent probably is that BASE_IND gets all characters with UISC = Other that haven’t been claimed for other USE classes already. The USE class OTHER uses “Any other SCRIPT_COMMON characters” to describe a similar fallback. However, any such description would have to clarify the classification of characters that have both Script Common and UISC Other. Or maybe the classes BASE_IND and OTHER can simply be merged into one? Is there any reason to keep them separate?
